### PR TITLE
add currently known veYFI specs

### LINKED ISSUE #214: Add YIP-65 to the docs (veYFI)
it's not live yet but worth documenting already since other yips that we want to document might depend on this one like #170 

https://gov.yearn.finance/t/yip-65-evolving-yfi-tokenomics/11994

### PR DESCRIPTION
**Pull request by MarcoWorms**
closes #214 

review at: https://yearn-docs-git-fork-marcoworms-veyfi.yearn.farm/contributing/governance/veyfi

### CODE REVIEW

File: docs/contributing/governance/veyfi.md
Comment (DarkGhost7) [inline docs/contributing/governance/veyfi.md:46]:
```suggestion
- **Incentivizes a **long-term** view on** Yearn.** Token holders are motivated to support the protocol over the long-term rather than to speculate in the short-term.
```

The bold here is messed up

File: docs/contributing/governance/veyfi.md
Comment (DarkGhost7) [inline docs/contributing/governance/veyfi.md:18]: You say it can be locked up to 4 years, then right below it you say it hasn't been determined. I would consolidate the two points into 1.

File: docs/contributing/governance/veyfi.md
Comment (DarkGhost7) [inline docs/contributing/governance/veyfi.md:19]:
```suggestion
- The maximum lock duration is still tbd, but will be in the range of min 1 year, max 4 years.
```
I wouldnt use shortened terms like this in the docs, best to write it out 'to be determined'

File: docs/contributing/governance/veyfi.md
Comment (DarkGhost7) [inline docs/contributing/governance/veyfi.md:35]:
```suggestion
  - if user has no lock = 0% of their rewards
```
maybe just say no rewards

File: docs/contributing/governance/veyfi.md
Comment (DarkGhost7) [inline docs/contributing/governance/veyfi.md:34]: of their rewards

File: docs/contributing/governance/veyfi.md
Comment (pandadefi) [inline docs/contributing/governance/veyfi.md:19]: Min is one week

File: docs/contributing/governance/veyfi.md
Comment (pandadefi) [inline docs/contributing/governance/veyfi.md:24]: The penalty is up to 75% and decaying overtime. max(75%, time remaining / 4 years)

File: docs/contributing/governance/veyfi.md
Comment (pandadefi) [inline docs/contributing/governance/veyfi.md:30]: bi-weekly

File: docs/contributing/governance/veyfi.md
Comment (pandadefi) [inline docs/contributing/governance/veyfi.md:31]: up to 10x

File: docs/contributing/governance/veyfi.md
Comment (pandadefi) [inline docs/contributing/governance/veyfi.md:32]: That and the following lines should be removed, it's not the case anymore.
Instead: A claim with a boost under 100% will send the leftover boost to the veYFI holders.

File: docs/contributing/governance/veyfi.md
Comment (MarcoWorms) [inline docs/contributing/governance/veyfi.md:3]: @saltyfacu do you think merging this PR would be an issue even with this disclaimer that the product is not live yet? Just pinging to be sure this PR stays on-hold :)

Comment (saltyfacu) [reply docs/contributing/governance/veyfi.md:3]: yeah, I'd like it to be finished. We can coord with the team building it. I'd also like someone else to review this for accuracy

File: docs/contributing/governance/veyfi.md
Comment (DarkGhost7) [inline docs/contributing/governance/veyfi.md:22]:
```suggestion
### Vault Gauges + Voting
```
Capitalize titles / headings

**Review by DarkGhost7 — CHANGES_REQUESTED** [review]
after this fix looks good to me great job on this

File: docs/contributing/governance/veyfi.md
Comment (DarkGhost7) [inline docs/contributing/governance/veyfi.md:36]:
```suggestion
- two or three weeks.
```
Hopefully we can get a definite number here before we merge this to the docs

**Comment by saltyfacu** [comment]
This is great! Please don't merge until veYFI is announced. If not people are going to ask how they vote if veYFI is not there yet.

File: docs/contributing/governance/veyfi.md
Comment (MarcoWorms) [inline docs/contributing/governance/veyfi.md:36]:
```suggestion
- You will be able to start claiming from the veYFI reward pool two or three weeks (to be defined) from the Thursday after which you lock before you can claim.
```

**Review by DarkGhost7 — APPROVED** [review]
lgtm

**Pull request merged**
